packages/vue/src/components/atoms/SfButton/SfButton.vue

```diff
@@ -5,6 +5,7 @@
     class="sf-button"
     v-bind="$attrs"
     :disabled="disabled"
+    :type="isButton"
```

Comment:
I don't think we need this additional prop - `type` is one of the built-in HTML element attribute for `button`, so may just use `$attrs` to see if any `type` passed. WDYT?

Reply:
you're right, I'll add type only for button in modal :)